setup.py

```diff
@@ -272,7 +272,7 @@ def setup_package():
         rxd_params = extension_common_params.copy()
         rxd_params['libraries'].append("rxdmath")
         rxd_params.update(dict(
-            extra_compile_args=["-O0"],  # cython files take too long to compile with O3
+            extra_compile_args=["-O2"],
```

Comment:
We should simply remove this flag so that other configurations may change this, like env vars

Reply:
to be tackled here : https://github.com/neuronsimulator/nrn/issues/518